pages/dkp/konvoy/2.2/choose-infrastructure/eks/quick-start/index.md

```diff
@@ -77,12 +77,12 @@ Tips:
     This creates a unique name every time you run it, so use the command with forethought.
 
     ```bash
-    CLUSTER_NAME=$(whoami)-aws-cluster-$(LC_CTYPE=C tr -dc 'a-z0-9' </dev/urandom | fold -w 5 | head -n1)
+    CLUSTER_NAME=$(whoami)-eks-cluster-$(LC_CTYPE=C LC_COLLATE=C tr -dc 'a-z0-9' < /dev/urandom | fold -w 5 | head -c 5)
```

Comment:
Lets keep this what it was before to be consistent with the other pages
```suggestion
    CLUSTER_NAME=$(whoami)-eks-cluster-$(LC_CTYPE=C tr -dc 'a-z0-9' </dev/urandom | fold -w 5 | head -n1)
```

Reply:
boo lol